Core/MIPS/ARM64/Arm64CompLoadStore.cpp

```diff
@@ -129,13 +129,11 @@ namespace MIPSComp {
 			}
 		}
 
-		DISABLE;
-
 		u32 iaddr = gpr.IsImm(rs) ? offset + gpr.GetImm(rs) : 0xFFFFFFFF;
 		std::vector<FixupBranch> skips;
 
 		if (gpr.IsImm(rs) && Memory::IsValidAddress(iaddr)) {
-			u32 addr = iaddr;
+			u32 addr = iaddr & 0x3FFFFFFF;
```

Comment:
Is this masking needed?

Reply:
Probably not, I just did a pass to make them all do the same thing.  At first I thought we had some platform where it was needed.

I think it was from copying the 32-bit armjit.  I'll just go ahead and remove all the masking.

-[Unknown]